demo/api/src/products/entities/raw-product.entity.ts

```diff
@@ -0,0 +1,119 @@
+import { Entity, Enum, ManyToOne, Property, Ref, types } from "@mikro-orm/core";
+import { CsvColumn } from "@src/importer/decorators/csv-column.decorator";
+import { FieldTransformer } from "@src/importer/decorators/field-transformer.decorator";
+import { TargetEntity } from "@src/importer/decorators/target-entity.decorator";
+import { BaseTargetEntity } from "@src/importer/entities/base-target.entity";
+import { Manufacturer } from "@src/products/entities/manufacturer.entity";
+import { IsArray, IsBoolean, IsDate, IsEnum, IsInt, IsOptional, IsString } from "class-validator";
+
+import { ProductDimensions, ProductDiscounts, ProductPriceRange, ProductStatus } from "./product.entity";
+import { ProductType } from "./product-type.enum";
+
+@Entity()
+@TargetEntity()
+export class RawProduct extends BaseTargetEntity {
```

Comment:
We don't need this entity in Demo. Put CsvColumn decorators directly on product entity